daemons/controld/controld_cib.c

```diff
@@ -909,3 +913,32 @@ controld_cib_delete_last_failure(const char *rsc_id, const char *node,
                                             cib_quorum_override|cib_xpath);
     free(xpath);
 }
+
+/*!
+ * \internal
+ * \brief Delete resource history entry from the CIB, given operation key
+ *
+ * \param[in] rsc_id     Name of resource to clear history for
+ * \param[in] node       Name of node to clear history for
+ * \param[in] key        Operation key of operation to clear history for
+ * \param[in] call_id    If specified, delete entry only if it has this call ID
```

Comment:
Excellent. Extra spaces between params and descriptions is the only thing I see, and I wouldn't bother with that since there's nothing else to change.